### PR TITLE
added The UnderLine links to the create Profile Page, 1020 

### LINKED ISSUE #1020: Add underline to Privacy Policy and Terms&Conditions to show that it is a clickable link
Lets add an underline to the Privacy Policy and Terms & Conditions links on the sign up ages for Create User Profile and Create organization profile. We should only underline the highlighted parts from the image below

image

### PR DESCRIPTION
**Pull request by Sridevi2014**
### What does this pull request aim to achieve? 💯
added The UnderLine links to the create Profile Page, 1020 
_Please be concise and link any related issue(s):_

 Resolves #1020

- [ ] I have checked that no one else is working on similar changes.
- [ ] I have read the latest [**README**](https://github.com/FightPandemics/FightPandemics/blob/master/README.md) and understand the development workflow.
- [ ] The name of this branch is: **`feature/<branch_name>`** (need to have cloned the repo not forked).
- [ ] This branch is rebased/merged with the **latest master**.
- [ ] There are no merge conflicts.
- [ ] There are no console warnings and errors.
- [ ] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".

### CODE REVIEW

File: client/src/pages/CreateOrganizationProfile.js [hunk 1]
```diff
@@ -39,6 +39,15 @@ import {
...
 import { inlineLabelStyles } from "constants/formStyles";
+import styled from "styled-components";
+
+
+const FooterLink = styled(Link)`
+
+   margin: 0.50em;
+  text-decoration-line: underline;
+
+`;
```
Comment (idjevm) [inline client/src/pages/CreateOrganizationProfile.js:50]:
```suggestion

const FooterLink = styled(Link)`
  margin: 0.50em;
  text-decoration-line: underline;
`;
```

File: client/src/pages/CreateOrganizationProfile.js [hunk 2]
```diff
@@ -301,27 +310,29 @@ const CreateOrgProfile = (props) => {
             <WhiteSpace />
             <WhiteSpace />
           </div>
+
```
Comment (idjevm) [inline client/src/pages/CreateOrganizationProfile.js:313]:
```suggestion
```

File: client/src/pages/CreateOrganizationProfile.js [hunk 3]
```diff
@@ -301,27 +310,29 @@ const CreateOrgProfile = (props) => {
             <WhiteSpace />
             <WhiteSpace />
           </div>
+          
           <InputWrapper>
-            <Label style={styleLabel} label="" />
+            <Label style={styleLabel} label="" />
```
Comment (idjevm) [inline client/src/pages/CreateOrganizationProfile.js:315]:
```suggestion
            <Label style={styleLabel} label="" />
```

File: client/src/pages/CreateOrganizationProfile.js [hunk 4]
```diff
@@ -301,27 +310,29 @@ const CreateOrgProfile = (props) => {
...
               value="I agree to the Terms and Conditions"
               onChange={handleInputChangeConditions}
-            >
+            >              
               By signing up, I agree to the{" "}
-              <Link to="/terms-conditions">Terms and Conditions</Link>
+              <FooterLink to="/terms-conditions">Terms and Conditions</FooterLink>
             </StyledCheckbox>
           </InputWrapper>
+
```
Comment (idjevm) [inline client/src/pages/CreateOrganizationProfile.js:335]:
```suggestion
```

File: client/src/pages/CreateOrganizationProfile.js [hunk 5]
```diff
@@ -301,27 +310,29 @@ const CreateOrgProfile = (props) => {
...
+              <FooterLink to="/privacy-policy">Privacy Policy</FooterLink>
             </StyledCheckbox>
             <WhiteSpace />
             <Label style={styleLabel} label="" />
             <StyledCheckbox
               style={{ fontSize: "1.2rem" }}
               value="I agree to the Terms and Conditions"
               onChange={handleInputChangeConditions}
-            >
+            >
```
Comment (idjevm) [inline client/src/pages/CreateOrganizationProfile.js:330]:
```suggestion
            >
```

File: client/src/pages/CreateUserProfile.js [hunk 6]
```diff
@@ -38,6 +38,15 @@ import { validateEmail } from "../utils/validators";
...
 import { SET_USER } from "../constants/action-types";
 
+
+
+const FooterLink = styled(Link)`
+
+   margin: 0.50em;
+  text-decoration-line: underline;
+
+`;
```
Comment (idjevm) [inline client/src/pages/CreateUserProfile.js:48]:
```suggestion
const FooterLink = styled(Link)`
  margin: 0.50em;
  text-decoration-line: underline;
`;
```

File: client/src/pages/CreateUserProfile.js [hunk 7]
```diff
@@ -131,6 +140,7 @@ const Submit = styled(SubmitButton)`
   font-weight: 500;
 `;
 
+
```
Comment (idjevm) [inline client/src/pages/CreateUserProfile.js:143]:
```suggestion
```

File: client/src/pages/CreateUserProfile.js [hunk 8]
```diff
@@ -391,7 +401,7 @@ const CreateProfile = ({ email, history }) => {
               onChange={handleInputChangeConditions}
             >
               By signing up, I agree to the{" "}
-              <Link to="/terms-conditions">Terms and Conditions</Link>
+              <FooterLink  to="/terms-conditions">Terms and Conditions</FooterLink >
```
Comment (idjevm) [inline client/src/pages/CreateUserProfile.js:404]:
```suggestion
              <FooterLink to="/terms-conditions">Terms and Conditions</FooterLink >
```

File: client/src/pages/CreateUserProfile.js [hunk 9]
```diff
@@ -382,7 +392,7 @@ const CreateProfile = ({ email, history }) => {
               onChange={handleInputChangePrivacy}
             >
               By signing up, I agree to the{" "}
-              <Link to="/privacy-policy">Privacy Policy</Link>
+              <FooterLink  to="/privacy-policy">Privacy Policy</FooterLink >
```
Comment (idjevm) [inline client/src/pages/CreateUserProfile.js:395]:
```suggestion
              <FooterLink to="/privacy-policy">Privacy Policy</FooterLink >
```

**Comment by Sridevi2014** [comment]
Deleted space

**Comment by rabbi08** [comment]
Validated #916 #917 
"Privacy Policy" and "Terms and conditions" are underlined and when clicked open in new TAB.

**Pull request merged**